### PR TITLE
Feedback: update fetch_feedback command line help

### PR DESCRIPTION
**Pull request by rkdarst**
- I don't really know if everthing in this description update is
  correct, but I did the best I could.  Please sanity-check first.
- As noticed by @sefriol

### CODE REVIEW

File: nbgrader/apps/fetchfeedbackapp.py [hunk 1]
```diff
@@ -26,25 +26,22 @@ class FetchFeedbackApp(NbGrader):
...
+        Fetch feedback for an assignment, if an instructor has released it.
         For the usage of students.
 
-        You can run this command from any directory, but usually, you will have a
-        directory where you are keeping your course assignments.
+        You can run this command from any directory, but usually, you will run
+        it from the directory where you are keeping your course assignments.
 
-        To fetch an assignment by name into the current directory:
+        To fetch feedback, you must first know the `course_id` for your course.
```
Comment (jhamrick) [inline nbgrader/apps/fetchfeedbackapp.py]: If the course_id is set in the `nbgrader_config.py` file, then you don't need to explicitly pass it as a flag, so I would maybe leave the current documentation that shows it without the flag.

**Comment by rkdarst** [comment]
Updated.  One thing I wasn't clear on is if it had to be run from the main notebook dir/assignment dir, or could be run anywhere.  The code wasn't completely clear to me last time I checked.

**Comment by Sefriol** [comment]
Atleast in our environtment you can run it in any folder. Feedback destination is always the same.

File: nbgrader/apps/fetchfeedbackapp.py [hunk 2]
```diff
@@ -26,25 +26,28 @@ class FetchFeedbackApp(NbGrader):
...
         For the usage of students.
 
-        You can run this command from any directory, but usually, you will have a
-        directory where you are keeping your course assignments.
+        You can run this command from any directory, but usually, you will run
+        it from the directory where you are keeping your course assignments.
 
         To fetch an assignment by name into the current directory:
 
             nbgrader fetch assignment1
```
Comment (jhamrick) [inline nbgrader/apps/fetchfeedbackapp.py]:
```suggestion
             nbgrader fetch_feedback assignment1
```

**Pull request merged**